src/components/forms/IdenticonField.js

```diff
@@ -10,13 +10,12 @@ import ShieldIdenticon from 'console/components/common/ShieldIdenticon';
 export default class IdenticonField extends React.PureComponent {
   static propTypes = {
     disabled: PropTypes.bool,
-    onChange: PropTypes.func.isRequired,
+    onChange: PropTypes.func,
     value: PropTypes.any,
   };
 
   static defaultProps = {
     disabled: false,
-    value: null,
```

Comment:
it sucks that this value is "implicitely passed down" from the antd component. It makes it difficult to track down its origin when debugging/maintaining :(

Reply:
yeah. forms in react have always been not the greatest.